src/components/keypad/button.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import {StyleSheet} from "aphrodite";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+
+type Props = {
+    onPress: () => void,
+    children: mixed,
+};
+
+const styles = StyleSheet.create({
```

Comment:
I'm curious about the button sizing, and whether the width, height should be an exported const. Will the keypad code that lays out the button require knowledge of the sizing? Or will the layout automatically flow?

Reply:
Sizing gets fixed up #64- the height is always constant but the width can change